pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -539,7 +539,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get").Groups(kapiGroup).Resources("pods/log").RuleOrDie(),
 				rbacv1helpers.NewRule("create", "list").Groups(kapiGroup).Resources("events").RuleOrDie(),
 
-				rbacv1helpers.NewRule("update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
+				rbacv1helpers.NewRule("create", "update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
```

Comment:
@openshift/sig-auth what did we get with 1.12 rebase that could have caused the need for this change? @bparees or did something changed wrt to imagestream SARs?

Reply:
we haven't messed with rbac recently. unsure what the cause is.

Reply:
@deads2k any chance you are aware of a connection to 1.12?

Reply:
@tnozicka I was looking into it, but we should not require that fix, the idea is that you have update role for istag and that allows you to create imagestream b/c we (the apiserver) do that for you. That's how @deads2k  explained me that (I'm missing the right wording here :man_shrugging: sorry). So this fix should not be needed and it should still work.

Reply:
@soltysh I get that creating IST means creating IS, I don't know why update on IST should imply create on IST - I actually think that the needed change is correct because update should not imply create - given some fields are immutable that would be elevating privileges. I guess all I wonder is how it worked till now.

Reply:
@smarterclayton master team has converged on moving forward with this change, as you might be the only one who knows why this worked so far - is this ok with you?